data/soapvalidator/Prefs/OutOfOffice/OOOPreventive.xml

```diff
@@ -0,0 +1,192 @@
+        
+        <t:test id="auth_account1" required="true">
+            <t:request>
+                <AuthRequest xmlns="urn:zimbraAccount">
+                    <account by="name">${account1.name}</account>
+                    <password>${defaultpassword.value}</password>
+                </AuthRequest>
+            </t:request>
+            <t:response>
+                <t:select path="//acct:AuthResponse/acct:authToken" set="authToken"/>
+            </t:response>
+        </t:test>
+        
+        <!-- Enable OOO with From and untilDate in the past -->
+        <t:test id="set_ooo_until_date" required="true" depends="auth_account1">
+            <t:request>
+                <ModifyPrefsRequest xmlns="urn:zimbraAccount">
+                    <pref name="zimbraPrefOutOfOfficeReplyEnabled">TRUE</pref>
+                    <pref name="zimbraPrefOutOfOfficeReply">${account1.reply}</pref>
+                    <pref name="zimbraPrefOutOfOfficeStatusAlertOnLogin">TRUE</pref>
+                    <pref name="zimbraPrefOutOfOfficeFromDate">20240827145900Z</pref>
+                    <pref name="zimbraPrefOutOfOfficeUntilDate">20240831145900Z</pref>
+                </ModifyPrefsRequest>
+            </t:request>
+            <t:response>
+                <t:select path="//acct:ModifyPrefsResponse"/>
+            </t:response>
+        </t:test>
+
+        <!-- Test 1: Verify OOO status is inactive after untilDate -->
+        <t:test id="check_ooo_inactive_after_date" required="true" depends="VerifyOOO_UntilDateReached">
+            <t:request>
+                <GetOutOfOfficeRequest xmlns="urn:zimbraMail">
+                    <email name="${account1.name}"/>
+                    <email name="${account2.name}"/>
+                </GetOutOfOfficeRequest>
+            </t:request>
+            <t:response>
+                <t:select path="//mail:GetOutOfOfficeResponse/mail:email[@name='${account1.name}'][@isOutOfOffice='0']"/>
+            </t:response>
+        </t:test>
+    </t:test_case>
+ 
+    <t:test_case testcaseid="Set_OOO_Future_2025" type="always" depends="AdminSetup">
+    <t:objective>OOO For Future 2025</t:objective>
+        <t:test id="set_ooo_message" required="true" depends="auth_account1">
+            <t:request>
+                <ModifyPrefsRequest xmlns="urn:zimbraAccount">
+                    <pref name="zimbraPrefOutOfOfficeReplyEnabled">TRUE</pref>
+                    <pref name="zimbraPrefOutOfOfficeReply">${account1.reply}</pref>
+                    <pref name="zimbraPrefOutOfOfficeStatusAlertOnLogin">TRUE</pref>
+                    <pref name="zimbraPrefOutOfOfficeUntilDate">20251231120000Z</pref>
+                </ModifyPrefsRequest>
+            </t:request>
+            <t:response>
+                <t:select path="//acct:ModifyPrefsResponse"/>
+            </t:response>
+        </t:test>
+    </t:test_case>
+
+    <!-- Test 2: Verify OOO Alert for Account 2 when adding Account 1 OOO Until set to Future -->
```

Comment:
Please merge test case Set_OOO_Future_2025 and VerifyOOOAlert as Set_OOO_Future_2025 is setup case for VerifyOOOAlert

Reply:
Merged in a single Test case